src/HomeNet/Data/Models/Identity.cs

```diff
@@ -12,39 +12,150 @@ namespace HomeNet.Data.Models
   /// </summary>
   public class Identity
   {
+    private static NLog.Logger log = NLog.LogManager.GetLogger("HomeNet.Data.Models.Identity");
+
+    /// <summary>Maximum number of bytes that identity name can occupy.</summary>
+    public const int MaxProfileNameLengthBytes = 64;
+
+    /// <summary>Maximum number of bytes that profile image can occupy.</summary>
+    public const int MaxProfileImageLengthBytes = 20 * 1024;
+
+    /// <summary>Maximum number of bytes that thumbnail image can occupy.</summary>
+    public const int MaxThumbnailImageLengthBytes = 5 * 1024;

```

Comment:
I'm not sure I understand the question. The maximal size is for file size. JPEG and PNG files are going to be supported. If you upload a file bigger than the limit, it will be refused.